drivers/adc/ad7616/ad7616.c

```diff
@@ -191,16 +195,16 @@ int32_t ad7616_spi_write(ad7616_dev *dev,
  * @param reg_data - The register data.
  * @return 0 in case of success, negative error code otherwise.
  */
-int32_t ad7616_par_read(ad7616_dev *dev,
-						uint8_t reg_addr,
-						uint16_t *reg_data)
+int32_t ad7616_par_read(struct ad7616_dev *dev,
```

Comment:
Does this change mean that the driver will support the specific HDL core for this device? Do we want the device driver to do this?

Reply:
The driver is dependent on the HDL core. (see also the old driver implementation: https://github.com/analogdevicesinc/no-OS/blob/d0bbfe561c64e539e5ab64dd8473f93847faa1db/ad7616-sdz/ad7616.c#L194-L226 )

This approach is meant to remove the ad7616_core layer, since the serial capture is already covered by the new spi_engine.
And for the parallel capture, the ad7616_core contains mainly AXI_IO read/write wrappers.